packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -622,13 +641,19 @@ class GutterItem extends StatelessWidget {
 
     final bpBoxSize = breakpointRadius * 2;
     final executionPointIndent = scaleByFontFactor(10.0);
-
+    Color? color;
+    if (coverageHit != null && isExecutable) {
```

Comment:
is the isExecutable check enough here? In your screenshot it looked like a line with only an end bracket } came back as executable (it was highlighted in red)

Reply:
Yeah, I don't think it is. I'll look at what Observatory does.

Reply:
Observatory is using some private fields to further refine what should be included in coverage information, but this change at least won't include the end bracket anymore.